hathor/feature_activation/storage/feature_activation_storage.py

```diff
@@ -0,0 +1,105 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from structlog import get_logger
+
+from hathor.conf.settings import HathorSettings
+from hathor.exception import InitializationError
+from hathor.feature_activation.feature import Feature
+from hathor.feature_activation.model.criteria import Criteria
+from hathor.feature_activation.settings import Settings as FeatureActivationSettings
+from hathor.storage import RocksDBStorage
+
+_CF_NAME_META = b'feature-activation-metadata'
+_KEY_SETTINGS = b'feature-activation-settings'
+
+logger = get_logger()
+
+
+class FeatureActivationStorage:
+    __slots__ = ('_log', '_settings', '_db', '_cf_meta')
+
+    def __init__(self, *, settings: HathorSettings, rocksdb_storage: RocksDBStorage) -> None:
+        self._log = logger.new()
+        self._settings = settings
+        self._db = rocksdb_storage.get_db()
+        self._cf_meta = rocksdb_storage.get_or_create_column_family(_CF_NAME_META)
```

Comment:
Should we use a single column family for all settings related stuff?

Reply:
Makes sense, but what other related stuff do you mean? Like using it to store the whole `HathorSettings` instance?

Reply:
Now the whole HathorSettings but we do have some flags stored in the database here and there in the code.

Reply:
I guess we don't have to do this refactor in this PR. So I'll approve this PR and we can discuss this refactor separately. Thanks!